src/bionics.cpp

```diff
@@ -935,8 +948,27 @@ void player::process_bionic( int b )
                                _( "You feel your throat open up and air filling your lungs!" ) );
             remove_effect( effect_asthma );
         }
-    } else if( bio.id == "afs_bio_dopamine_stimulators" ) { // Aftershock
+    } else if( bio.id ==
+               "afs_bio_dopamine_stimulators" ) { // Aftershock-specific bionic; gives a constant mood boost
         add_morale( MORALE_FEELING_GOOD, 20, 20, 30_minutes, 20_minutes, true );
+    } else if( bio.id ==
+               "afs_bio_lightning_rod" ) { // Aftershock-specific bionic; charges power through lightning
+        if( g->weather.lightning_active && g->m.is_outside( pos() ) && one_in( 3 ) ) {
+            g->weather.lightning_active = false;
+            sounds::sound( pos(), 300, sounds::sound_t::combat, _( "BOOOOOOOOOOOOOOOOOOM!!!!" ) );
+            add_msg_if_player( " " );
+            add_msg_if_player( m_mixed, _( "Lightning strikes your lightning rod!" ) );
+            add_msg_if_player( m_good, _( "Your bionics surge with energy!  Your nerves scream with power!" ) );
+            add_msg_if_player( " " );
+            g->u.add_effect( effect_blind, rng( 1_minutes, 2_minutes ) );
```

Comment:
```suggestion
            add_effect( effect_blind, rng( 1_minutes, 2_minutes ) );
```
No need to drag `g->u` in here. This might be called on an NPC.

Btw. we *did* you use `g->u` here, but *one* line below, you did not?

Reply:
Call it a misstep, I'm pretty rusty. I'll patch it up.